src/command_modules/azure-cli-documentdb/HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.1.5 (2017-04-28)
+++++++++++++++++++
```

Comment:
Instead of the date, please change to `unreleased`. The date will be completed when it is actually released.